proposals/dao/fip_82.ts

```diff
@@ -69,8 +71,7 @@ const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: Named
   );
   await podAdminGateway.deployTransaction.wait();
   logging && console.log(`Deployed PodAdminGateway at ${podAdminGateway.address}`);
-  await transferOrcaTokens(addresses.orcaShipToken, deployAddress, podFactory.address, 2);
-  logging && console.log('Pod factory deployed to:', podFactory.address);
+  await transferOrcaTokens(addresses.orcaShipToken, deployAddress, podFactory.address, 1);
```

Comment:
Just want to verify here that the change from `number: 2` to `number: 1` was deliberate

Reply:
Yep, SHIP token is a permissioning method Orca uses for deploying new pods. Only have a limited supply atm, so didn't want to send a lot out in case the deploy was incorrect and needed to be redone